src/program.cpp

```diff
@@ -605,13 +606,26 @@ std::vector<argument> program::eval(parameter_map params, execution_environment
     return ret;
 }
 
-const int program_file_version = 5;
+std::string get_migraphx_version()
+{
+    std::stringstream ss;
+    ss << std::to_string(MIGRAPHX_VERSION_MAJOR) << "." << std::to_string(MIGRAPHX_VERSION_MINOR)
+       << "." << std::to_string(MIGRAPHX_VERSION_PATCH);
+    return ss.str();
+}
+
+/*
+program file version is for the data structure or format of the MXR file. Version should be bumped
+if any changes occur to the format of the MXR file.
+*/
+const int program_file_version = 6;
 
 value program::to_value() const
 {
     value result;
-    result["version"] = program_file_version;
-    result["target"]  = this->impl->target_name;
+    result["version"]      = program_file_version;
+    result["migx_version"] = get_migraphx_version();
```

Comment:
I think it would be better for it be `migraphx_version`.

Reply:
Done.